daisy_workflows/export/export_disk.sh

```diff
@@ -13,12 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+BYTES_1GB=1073741824
 URL="http://metadata/computeMetadata/v1/instance/attributes"
 GCS_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/gcs-path)
 LICENSES=$(curl -f -H Metadata-Flavor:Google ${URL}/licenses)
 
 mkdir ~/upload
 
+# Source disk size info.
+SOURCE_SIZE_BYTES=$(lsblk /dev/sdb --output=size -b | sed -n 2p)
+SOURCE_SIZE_GB=$(awk "BEGIN {print int(((${SOURCE_SIZE_BYTES}-1)/${BYTES_1GB}) + 1)}")
+echo "GCEExport: <serial-output key:'source-size-gb' value:'${SOURCE_SIZE_GB}'>"
```

Comment:
Extract <serial-output> formatting to a func to avoid code duplication and reduce introducing errors.

In other scripts as well